ci/build-mingw64.sh

```diff
@@ -271,7 +271,7 @@ rm -rf $build
 meson setup $build --cross-file "$prefix_dir/crossfile" \
     --buildtype debugoptimized \
     -Dlibmpv=true -Dlua=luajit \
-    -D{shaderc,spirv-cross,d3d11,libplacebo}=enabled
+    -D{shaderc,spirv-cross,d3d11}=enabled
```

Comment:
You can also remove all `libplacebo` custom build from this script, use wrap instead.

Reply:
we set nodownload to wrap above, and removing nodownload breaks libass/freetype build so I think I'll just leave it there